scripts/graphql-codegen-override-pv.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs')
```

Comment:
this file is a simple string replace on the generated graphql types file, changing pv properties from `string` to `PV`